internal/command/init.go

```diff
@@ -130,3 +147,17 @@ func init() {
 
 	rootCmd.AddCommand(initCmd)
 }
+
+func cleanComposeProjectName(input string) string {
```

Comment:
I'm missing why we need this function, can't we just use a regex here?

Reply:
It's because when we first init the project, we take the project name from the current directory. We don't want to throw errors if the directory name is "weird", so we instead "clean" it following the same algorithm that compose does normally by stripping out invalid characters.

If we just used a regex we'd only be able to throw an error which would make a bad UX for users of the init subcommand.